DNA_app.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+import streamlit as st
+import altair as alt 
+from PIL import Image 
+
```

Comment:
Please review this once again!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
thanks